contracts/pass-sale/CruzoPassSale.sol

```diff
@@ -0,0 +1,120 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+import "../tokens/Cruzo1155.sol";
+
+import "../utils/Cruzo1155Factory.sol";
+
+contract CruzoPassSale is Ownable {
+    uint256 public constant MAX_PER_ACCOUNT = 3;
```

Comment:
@kewka Would you mind making these values `immutable` instead of `constant` and assign its values in the constructor? This way we can define it in code just once and at the same time the contract should remain gas efficient.

Reply:
constants should be cheaper than immutable variables